tests/Admin/BaseAdminModelManagerTest.php

```diff
@@ -81,7 +83,7 @@ public function testId(): void
             ->method('getNormalizedIdentifier')
             ->willReturn('42');
 
-        $admin = new BaseAdminModelManager_Admin('code', \stdClass::class, 'controller');
+        $admin = new BaseAdminModelManager_Admin();
```

Comment:
Should we add / leave some test with group legacy in order to ensure we dont break old behavior?

Reply:
There is a group legacy for the constructor and some in the AddDependencyCallTest, this seems enough to me.